src/org/usfirst/frc4904/robot/subsystems/LIDAR.java

```diff
@@ -1,20 +1,46 @@
 package org.usfirst.frc4904.robot.subsystems;
 
 
-import org.usfirst.frc4904.standard.custom.motioncontrollers.MotionController;
+import org.usfirst.frc4904.standard.custom.motioncontrollers.CustomPIDController;
+import org.usfirst.frc4904.standard.custom.sensors.CANEncoder;
+import org.usfirst.frc4904.standard.custom.sensors.CANSensor;
 import org.usfirst.frc4904.standard.subsystems.motor.VelocitySensorMotor;
+import edu.wpi.first.wpilibj.PIDSourceType;
 import edu.wpi.first.wpilibj.SpeedController;
 
-public class LIDAR extends VelocitySensorMotor {
+public class LIDAR {
 	public static double MIN_MOTOR_OUTPUT = 0.15;
 	public static double MAX_MOTOR_OUTPUT = 0.4;
 	public static double TURN_P = 0.000006; // WIP
 	public static double TURN_I = 0.00000000605; // WIP
 	public static double TURN_D = 0; // WIP
 	public static final double TURN_F = 0.00000138;
 	public static final double TARGET_MRPM = 240000;
+	protected final VelocitySensorMotor lidarMotor;
+	protected final CustomPIDController lidarMotorController;
+	protected final CANSensor lidarPhase1Sensor;
+	protected final CANSensor lidarPhase2Sensor;
 
-	public LIDAR(SpeedController motor, MotionController lidarController) {
-		super("LIDAR", lidarController, motor);
+	public LIDAR(SpeedController motor, int lidarPhase1ID, int lidarPhase2ID, int lidarEncoderID) {
```

Comment:
It seems like there's some duplicated code here from the `lidar` branch - is that OK or should we change the base branch of this PR to `lidar`?